### PR TITLE
RecordELResolver FAT

### LINKED ISSUE #27123: Expression Language 6.0: Add support for java.lang.Record instances. The support is enabled by default. Support is provided via the new RecordELResolver.
`https://github.com/jakartaee/expression-language/issues/201`

### PR DESCRIPTION
**Pull request by pnicolucci**
fixes #27123

### CODE REVIEW

**Comment by pnicolucci** [comment]
#libby #build

**Comment by pnicolucci** [comment]
#libby #build

File: dev/io.openliberty.org.apache.jasper.expressionLanguage.6.0_fat/test-applications/EL60RecordELResolverTest.war/src/io/openliberty/el60/fat/recordelresolver/servlets/EL60RecordELResolverTestServlet.java [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+public class EL60RecordELResolverTestServlet extends FATServlet {
+    private static final long serialVersionUID = 1L;
+    ELProcessor elp = new ELProcessor();
+
+    public EL60RecordELResolverTestServlet() {
+        elp.defineBean("testRecordString", new TestRecordString("Test Data!"));
+    }
+
+    /**
+     * Test the RecordELResolver for a String."
```
Comment (volosied) [inline dev/io.openliberty.org.apache.jasper.expressionLanguage.6.0_fat/test-applications/EL60RecordELResolverTest.war/src/io/openliberty/el60/fat/recordelresolver/servlets/EL60RecordELResolverTestServlet.java]: Single quotation mark at the end.

**Review by volosied — DISMISSED** [review]
Tests look fine to me

**Comment by pnicolucci** [comment]
#libby #build

**Review by volosied — APPROVED** [review]
Looks good

**Pull request merged**